client/tests/register_test.py

```diff
@@ -41,6 +42,25 @@ def test_sql_transformation_decorator_invalid_fn(local, fn):
     with pytest.raises((TypeError, ValueError)):
         decorator(fn)
 
+
+
+def test_sql_transformation_empty_description(registrar):
+    def my_function():
+        return "SELECT * FROM X"
+
+    dec = SQLTransformationDecorator(registrar=registrar, owner="", provider="", variant="sql")
+    dec.__call__(my_function)
+    dec.to_source()
+
+def test_df_transformation_empty_description(registrar):
+    def my_function(df):
+        return df
+
+    dec = DFTransformationDecorator(registrar=registrar, owner="", provider="", variant="df")
+    dec.__call__(my_function)
+    dec.to_source()
```

Comment:
Should this actually check something, if not put a comment specifying that we're testing that an error isn't thrown

Reply:
Yeah it checks that the definition can be converted to a source. Added comment